label_studio/projects/api.py

```diff
@@ -613,6 +613,7 @@ class TemplateListAPI(generics.ListAPIView):
     swagger_schema = None
 
     def list(self, request, *args, **kwargs):
+        data = request.data
```

Comment:
nit: since data isn't used anywhere

```suggestion
        _ = request.data
```

Reply:
This is just some pep styling I've seen else where, idk what the conventions are for this project, so do what you will.